configure-clusters/src/main/java/com/esri/arcgismaps/sample/configureclusters/screens/MainScreen.kt

```diff
@@ -0,0 +1,413 @@
+    clusterMaxScaleOptions: List<Int>,
+    clusterMaxScale: Int,
+    updateClusterMaxScaleState: (Int) -> Unit,
+    mapScale: Int
+) {
+    ModalBottomSheet(
+        sheetState = controlsBottomSheetState,
+        onDismissRequest = {
+            showControlsBottomSheet(false)
+        }) {
+        Row {
+            Column {
+                Text(
+                    "Cluster labels visibility:",
+                    modifier = Modifier.padding(8.dp)
+                )
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
+                        "Show labels",
+                        modifier = Modifier.padding(8.dp)
+                    )
+                    Switch(
+                        checked = showClusterLabels,
+                        onCheckedChange = { showClusterLabels ->
+                            updateClusterLabelState(
+                                showClusterLabels
+                            )
+                        },
+                        modifier = Modifier.padding(8.dp)
+                    )
+                }
+            }
+            Column {
+                Row {
+                    Spacer(modifier = Modifier.weight(1f))
+                    Box(
+                        modifier = Modifier
+                            .padding(8.dp)
+                            .background(Color.LightGray, MaterialTheme.shapes.medium)
+                    ) {
+                        Text(
+                            "Current map \nscale: 1:${mapScale}",
+                            modifier = Modifier.padding(8.dp),
+                            color = Color.Black
+                        )
+                    }
+                }
+            }
+        }
+        Row {
+            Column {
+                Divider(
+                    color = Color.LightGray,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .width(2.dp)
+                )
+                ClusterRadiusControls(
+                    clusterRadiusOptions,
+                    clusterRadius,
+                    updateClusterRadiusState
+                )
+                ClusterMaxScaleControls(
+                    clusterMaxScaleOptions,
+                    clusterMaxScale,
+                    updateClusterMaxScaleState
+                )
+                Spacer(modifier = Modifier.size(40.dp))
+            }
+        }
+    }
+}
+
+/**
+ * Composable function to display the cluster radius controls within the cluster controls bottom
+ * sheet.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun ClusterRadiusControls(
+    clusterRadiusOptions: List<Int>,
+    clusterRadius: Int,
+    updateClusterRadius: (Int) -> Unit
+) {
+    Text(
+        "Clustering properties:",
+        modifier = Modifier.padding(8.dp)
+    )
+    Row(verticalAlignment = Alignment.CenterVertically) {
```

Comment:
To horizontally space the two items in the row to take up the BottomSheet width you can instead use a `horizontalArrangement` here: 
```suggestion
    Row(
        Modifier.fillMaxWidth(),
        verticalAlignment = Alignment.CenterVertically,
        horizontalArrangement = Arrangement.SpaceBetween
    ) {
```